xhistogram/duck_array_ops.py

```diff
@@ -25,8 +25,8 @@ def f(*args, **kwargs):
 
     else:
 
-        def f(data, *args, **kwargs):
-            return getattr(eager_module, name)(data, *args, **kwargs)
+        def f(*args, **kwargs):
+            return getattr(eager_module, name)(*args, **kwargs)
```

Comment:
Does this signature need to include `data` as the first argument? If so, we might want to change these function names to `f_eager` and `f_dask` to make `mypy` happy

Reply:
I don't _think_ so...

I've just taken a look at xarray (where this function was originally copied from) and it looks like they made the same change as I have when they introduced mypy - see https://github.com/pydata/xarray/commit/6963164dbdc55f86f34b741073838f083a68a152.

Reply:
Well if it's good enough for xarray, then I think we're safe to do the same here : ) 

Also, looking at this a second time, since we're just using `*args` and just passing positional arguments along it definitely shouldn't matter whether or not we have `data` as the first argument